### ISSUE TITLE
Hm...taste like Kakao

### ISSUE THREAD

**Issue by Vacxe** [hyperdevs-team/morsa#2]
Have a good day. @adriangl @FrangSierra appreciate for your project. But honestly it looks like a https://github.com/KakaoCup/Compose. Based on our experience with https://github.com/KakaoCup/Kakao we added similar framework for Compose support.

**Comment by Vacxe** [comment]
Hm, I checked your readme, thanks for the reference. However you welcome to contribute for Kakaocup/Compose

**Comment by adriangl** [comment]
Hello @Vacxe!

Yes, as we mention in the README of the library we built this based on the syntax of https://github.com/KakaoCup/Kakao, so thanks for providing such an awesome library!

We currently use it in our projects, but we recently started migrating some of our apps' UI to Compose and we weren't aware of when Compose support was going to be added (apart from this issue in the archived Kakao repo: https://github.com/agoda-com/Kakao/issues/244). That's why we decided to build our own lib with similar APIs and share it with everyone that needed it.
Sorry that we didn't notice that Compose support was being worked on in another repo, our bad.

And yeah! Not sure if we'll be able to contribute much but we can contribute to the Kakao repo if we find something that we can do :)
